internal/playwright/config.go

```diff
@@ -126,6 +129,29 @@ func SetDefaults(p *Project) {
 			}
 		}
 	}
+
+	p.Suites = shardSuites(p.Suites)
+}
+
+// shardSuites applies sharding by replacing the original suites with the appropriate number of replicas according to
+// the numShards setting on each suite. A suite is only sharded if numShards > 1.
+func shardSuites(suites []Suite) []Suite {
+	var shardedSuites []Suite
+	for _, s := range suites {
+		// Use the original suite if there is nothing to shard.
+		if s.NumShards < 1 {
```

Comment:
Should this be that ?
```suggestion
		if s.NumShards <= 1 {
```

Reply:
You are right! I'll add a test for that one 👍